src/app/layout.tsx

```diff
@@ -1,12 +1,10 @@
+import { Footer, Navbar } from '../../components'
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Why do you need Bootstrap if you have Tailwind already? Any special reason?

Reply:
Offers already pre-done components like lets say MaterialUI, but we'll see how much it will actually be used in the end because I already changed one over the weekend so.